instrumentation/runtime-metrics/runtime-metrics-java8/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/java8/JmxRuntimeMetricsUtil.java

```diff
@@ -12,10 +12,9 @@
 import java.util.List;
 import javax.annotation.Nullable;
 
-class JmxRuntimeMetricsUtil {
+public class JmxRuntimeMetricsUtil {
```

Comment:
Can you move it to an `.internal` package?

Reply:
Ok I've moved it to `io.opentelemetry.instrumentation.runtimemetrics.java8.internal` now